### PR TITLE
Add existing public dashboards

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

**Review by TheoBrigitte — COMMENTED** [review]
Sounds like a descent setup but where are the actual dashboards json files ?

**Comment by QuentinBisson** [comment]
I forgot them somewhere :(

**Comment by TheoBrigitte** [comment]
I would also appreciate if you could split this PR, there are a lot of automatically generated files that you could just merge without any approval, because well they are generated so we trust them. They just add noise here.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,93 @@
...
+
+orbs:
+  architect: giantswarm/architect@2.7.0
+
 jobs:
-  build:
+  build-and-deploy-dashboards:
     docker:
-      - image: circleci/golang:1.15.0
+    - image: circleci/golang:1.15.0
```
Comment (mig4) [inline .circleci/config.yml:9]: ??

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,15 +1,93 @@
...
-            fi
+    - run: go get github.com/google/go-jsonnet/cmd/jsonnet
+    - run: git clone https://github.com/grafana/grafonnet-lib.git $GOPATH/src/github.com/grafana/grafonnet-lib
+    - checkout
+    - run: ./scripts/make-dashboards.sh
+    - run:
+        command: |
+          if [ "${CIRCLE_BRANCH}" == "master" ]; then
+            ./scripts/upload-dashboards.sh
+          fi
```
Comment (mig4) [inline .circleci/config.yml:19]: Please don't change the indentation.

**Review by mig4 — CHANGES_REQUESTED** [review]
Agreed, please split this, I have no idea what's what here, most of the changes don't seem to be related to the title of this PR.

**Comment by QuentinBisson** [comment]
I asked for review a bit too soon :(

**Comment by mig4** [comment]
> I asked for review a bit too soon :(

What I usually do is first submit as a draft PR without adding reviewers, give it a glance over myself, and then click _Ready for review_ and add reviewers. In case that comes in handy.

**Comment by QuentinBisson** [comment]
Tested on Gauss

File: LICENSE [hunk 3]
```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```
Comment (mig4) [inline LICENSE:201]: I can't tell why this shows up in the diff. Could you check there's no weird non-printable characters here or sth?

File: helm/dashboards/templates/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}  
+  name: grafana-dashboards-public-vmware
+  namespace: {{ .Values.namespace  }}
+data:
+{{ (.Files.Glob "dashboards/vmware/*").AsConfig | indent 2 }}
+---
+{{- end }}
```
Comment (mig4) [inline helm/dashboards/templates/configmap.yaml:68]: Any chance you could add a newline at the end of file?

**Review by mig4 — CHANGES_REQUESTED** [review]
I don't know, this is still a bit confusing, as far as I can see the changes are to add a helm chart to the repo?

File: LICENSE (hunk 3 above)
Comment (QuentinBisson) [reply LICENSE:201]: I think I just added a new line

**Comment by QuentinBisson** [comment]
I;m not really sure what is confusing here though, The helm chart change is straightforward to me

**Review by TheoBrigitte — APPROVED** [review]
Looks alright

**Pull request merged**